<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tienda Belleza</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frameForm">
      <property name="minimumWidth">
       <number>320</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutForm">
       <item>
        <widget class="QLabel" name="labelTitle">
         <property name="text"><string>Agregar producto</string></property>
         <property name="alignment"><set>Qt::AlignCenter</set></property>
        </widget>
       </item>
       <item>
        <widget class="QFormLayout" name="formLayout">
         <property name="labelAlignment"><set>Qt::AlignRight</set></property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelName"><property name="text"><string>Nombre:</string></property></widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPrice"><property name="text"><string>Precio:</string></property></widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinPrice">
           <property name="maximum"><double>1000000.0</double></property>
           <property name="decimals"><number>2</number></property>
           <property name="prefix"><string>$</string></property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelType"><property name="text"><string>Tipo:</string></property></widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineType"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelExpiry"><property name="text"><string>Caducidad:</string></property></widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateEdit" name="dateExpiry">
           <property name="calendarPopup"><bool>true</bool></property>
           <property name="date"><date>2025-01-01</date></property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelImage"><property name="text"><string>Imagen:</string></property></widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="hLayoutImage">
           <item>
            <widget class="QLineEdit" name="lineImage"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnSelectImage">
             <property name="text"><string>...</string></property>
            </widget>
           </item>
          </layout>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hButtons">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text"><string>Agregar</string></property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text"><string>Eliminar</string></property>
           <property name="enabled"><bool>false</bool></property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblPreview" >
         <property name="minimumHeight"><number>120</number></property>
         <property name="frameShape"><enum>QFrame::Box</enum></property>
         <property name="alignment"><set>Qt::AlignCenter</set></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutRight">
      <item>
       <layout class="QHBoxLayout" name="hFilter">
        <item>
         <widget class="QLabel" name="labelFilter"><property name="text"><string>Filtrar por tipo:</string></property></widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFilter"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableProducts"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>